wcivf/apps/elections/models.py

```diff
@@ -468,6 +469,23 @@ def next_ballot(self):
         except PostElection.DoesNotExist:
             return None
 
+    @property
+    def party_ballot_count(self):
+        if self.personpost_set.exists():
+            people = self.personpost_set
+            if self.election.uses_lists:
+                ind_candidates = people.filter(party_id="ynmp-party:2").count()
+                num_other_parties = (
+                    people.values("party_id").distinct().count()
+                    - ind_candidates
+                )
+                total_parties = ind_candidates + num_other_parties
+                return f"{total_parties} ballot options"
+            else:
+                return f"{people.count()} candidate{pluralize(people.count())}"
+        else:
+            return f"{self.election.name}: No candidates known yet."
```

Comment:
Just to make sure - when no candidates are known will this show as `You can choose from {election.name}: No candidates known yet in the {{ postelection.friendly_name }}`?

I don't really think this conditional is useful here - maybe we should just `return None` if we can't tell someone how many choices they will have? I expect this logic is already in the template anyway, so let's reproduce it here.

If we should return something, then we should make the sentence structure the same as the other returned strings